packages/visual-unit/src/helper/grid-lines.js

```diff
@@ -178,29 +172,30 @@ export const attachDataToGridLineLayers = (context) => {
         width: context.width(),
         height: context.height()
     };
-
+    const gridLines = context._gridLineSelection.getObjects();
+    const gridBands = context._gridBandSelection.getObjects();
     const gridLayerData = getGridLayerData(axes, context.fields(), context.data().getFieldsConfig());
-    [].concat(...context._gridbands, ...context._gridlines).forEach((inst) => {
+    [].concat(...gridBands, ...gridLines).forEach((inst) => {
         inst.data(inst.axes().x ? gridLayerData.x : gridLayerData.y).measurement(measurement);
     });
 };
 
 export const renderGridLineLayers = (context, container) => {
-    const axes = context.axes();
     const config = context.config();
     const classPrefix = config.classPrefix;
+    const gridLines = context._gridLineSelection.getObjects();
+    const gridBands = context._gridBandSelection.getObjects();
 
-    if (axes && ((axes.x && axes.x.length) || (axes.y && axes.y.length))) {
-        [[context._gridlines, `${classPrefix}-${GRIDLINEPARENTGROUPCLASS}`],
-            [context._gridbands, `${classPrefix}-${GRIDBANDPARENTGROUPCLASS}`]].forEach((entry) => {
+    const gridLineParentGroup = makeElement(container, 'g', [1], `${classPrefix}-${GRIDPARENTGROUP}`);
```

Comment:
it would be better if enums like GRIDBANDPARENTGROUPCLASS be written as GRID_BAND_PARENT_GROUP_CLASS. 
It would easier to read and would maintain consistency